plugin/src/main/resources/seedJobGroovyScript.groovy

```diff
@@ -119,7 +119,8 @@ def updateJobs(FilePath filePath, String namePrefix = '') {
 def workspace = build.workspace
 def xmlBase = new FilePath(workspace, 'XML_BASE_DIR_PLACEHOLDER')
 
-def localFile = new File("TEMPORARY_XML_DIR_PREFIX_PLACEHOLDER-${build.getNumber()}")
+def randomPostfixToAvoidCollisions = Math.abs(new Random().nextInt())
+def localFile = new File("TEMPORARY_XML_DIR_PREFIX_PLACEHOLDER-${build.getNumber()}-${randomPostfixToAvoidCollisions}")
```

Comment:
I'd suggest to use the `createTempDirectory` function from [`Files`](https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#createTempDirectory-java.lang.String-java.nio.file.attribute.FileAttribute...-) instead which takes care of generating an unused name:

```
import java.nio.file.Files

...

def localFile = Files.createTempDirectory("TEMPORARY_XML_DIR_PREFIX_PLACEHOLDER")
```

The `localFile.exists()` and `localFile.mkdir()` below could then be removed.

Reply:
Ok makes sense at this point.

Reply:
I'll get to this after my holiday in a week.